prebuild_binaries/linux_build_all.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+# Copyright 2018 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Fail on any error.
+set -e pipefail
+
+# Display commands
+set -x
+
+case $KOKORO_JOB_TYPE in
+  CONTINUOUS_GITHUB)
+    BUILD_TYPE=continuous
+    ;;
+  PRESUBMIT_GITHUB)
+    BUILD_TYPE=presubmit
+    ;;
+  RELEASE)
```

Comment:
This file appears to be referencing cloud-profiler-e2e service account key, so I wouldn't expect it to handle the release process? Where the final release binaries could to be published? gs://cloud-profiler bucket? That one is not cloud-profiler-e2e project, so it shouldn't use cloud-profiler-e2e service account.

Reply:
Done.

Reply:
I'm still seeing both
```
  RELEASE)
    BUILD_TYPE=release
    ;;
```
and
```
SERVICE_KEY="${KOKORO_KEYSTORE_DIR}/72935_cloud-profiler-e2e-service-account-key"
```
in the prebuild_binaries/linux_build_and_test.sh file so I am not sure it's done.

Reply:
The Java agent's release process uses this service key for running tests, but doesn't seem to specify a service key for copying files to the bucket in the release.

I think this should work.